include/mcmini/misc/MCSortedStack.hpp

```diff
@@ -4,6 +4,8 @@
 #include <stack>
 #include <stdexcept>
 
+// FIXME:  The only instantiation of MCSortedStack is for Index = unint32.
```

Comment:
It is indeed only used once. You can remove the template

Reply:
Fixed.